hackathon_site/dashboard/frontend/src/components/inventory/ProductOverview/ProductOverview.js

```diff
@@ -115,61 +131,44 @@ const DetailInfoSection = ({
-            <div className={styles.bodyinfo}>
-                <Typography variant="body2" className={styles.heading}>
-                    Constraints
-                </Typography>
-                {constraints.map((constraint, i) => (
-                    <Typography key={i} variant="body2">
-                        {constraint}
-                    </Typography>
-                ))}
-            </div>
-            <div className={styles.bodyinfo}>
-                <Typography variant="body2" className={styles.heading}>
-                    Manufacturer
-                </Typography>
-                <Typography>{manufacturer}</Typography>
-            </div>
-            <div className={styles.bodyinfo}>
-                <Typography variant="body2" className={styles.heading}>
-                    Model Number
-                </Typography>
-                <Typography>{model_num}</Typography>
-            </div>
-            <div className={styles.bodyinfo}>
-                <Typography variant="body2" className={styles.heading}>
-                    Datasheet
-                </Typography>
-                <Link
-                    href={datasheet}
-                    rel="noopener"
-                    color="inherit"
-                    underline="none"
-                    target="_blank"
-                    className={styles.bodyinfoDataSheet}
-                >
-                    <LaunchIcon />
-                    <Typography>Link</Typography>
-                </Link>
-            </div>
-            <div className={styles.bodyinfo}>
-                <Typography variant="body2" className={styles.heading}>
-                    Notes
-                </Typography>
-                <Typography variant="body2">{notes}</Typography>
-            </div>
-        </div>
+        <>
+            <Typography variant="body2" color="secondary" className={styles.heading}>
+                Constraints
+            </Typography>
+            {constraints.map((constraint, i) => (
+                <Typography key={i}>{constraint}</Typography>
+            ))}
+            <Typography variant="body2" className={styles.heading}>
+                Manufacturer
+            </Typography>
+            <Typography>{manufacturer}</Typography>
+            <Typography variant="body2" className={styles.heading}>
+                Model Number
+            </Typography>
+            <Typography>{model_num}</Typography>
+            <Typography variant="body2" className={styles.heading}>
+                Datasheet
+            </Typography>
+            <Button
+                href={datasheet}
+                rel="noopener"
+                target="_blank"
+                startIcon={<LaunchIcon />}
+            >
+                Link
+            </Button>
+            <Typography variant="body2" className={styles.heading}>
+                Notes
+            </Typography>
+            <Typography>{notes}</Typography>
```

Comment:
Since the notes can have new lines in them, you could make this 

```javascript
{notes.split("\n").map((note, i) => (
     <Typography key={i}>{note}</Typography>
))}
```

otherwise it won't listen to the `\n` characters.